Source/Scene/Cesium3DTile.js

```diff
@@ -1320,10 +1320,11 @@ import TileOrientedBoundingBox from './TileOrientedBoundingBox.js';
      *
      * @private
      */
-    Cesium3DTile.prototype.update = function(tileset, frameState) {
+    Cesium3DTile.prototype.update = function(tileset, frameState, passOptions) {
+        console.log(passOptions);
```

Comment:
I was noticing absolutely terrible FPS and got scared that this was one of those "fix a bug, hurt performance" issues, but then I realized this stray console.log is destroying Sandcastle.

Reply:
Sorry should have caught that. Fixed now.